src/layout/components/page-top.js

```diff
@@ -15,6 +19,29 @@ export class PageTop extends React.Component {
     }),
   }
 
+  constructor(props) {
+    super(props);
+    this.state = { messages: [{
+      user: {
+        name: 'Ashley',
+        picture: 'http://ww4.msu.ac.zw/mainsite/wp-content/uploads/2015/05/default.gif',
```

Comment:
Can we find an open source image and add it to our images folder, then import it at the top and use the imported variable here instead of the url? I don't know if this is open source.